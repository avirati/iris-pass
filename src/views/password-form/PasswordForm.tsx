import React from 'react'

import { ContentContainer } from 'components/content-container'
import { Button, Form, InputField } from 'shared-components'

const onSubmit = (value: any) => {
  console.log(value)
}

export const PasswordForm: React.FC = () => {
  return (
    <ContentContainer>
      <Form onSubmit={onSubmit} css={{ display: 'flex', gap: '$4', flexDirection: 'column' }}>
        <InputField label='Website' name='website' placeholder='Website (e.g. https://google.com)'/>
        <InputField label='Login' name='login' placeholder='Login (e.g. you@example.com)'/>
        <Button type="submit">Submit</Button>
      </Form>
    </ContentContainer>
  )
}