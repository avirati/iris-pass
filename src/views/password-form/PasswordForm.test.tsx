import { render } from '@testing-library/react'

import { PasswordForm } from './PasswordForm'

describe('PasswordForm Component', () => {
  it('renders', () => {
    const { container } = render(<PasswordForm />)
    expect(container).toMatchSnapshot()
  })
})