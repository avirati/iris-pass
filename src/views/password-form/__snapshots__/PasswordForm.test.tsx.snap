// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PasswordForm Component renders 1`] = `
<div>
  <div
    class="c-dhzjXW c-dhzjXW-ihydgyy-css"
  >
    <form
      aria-label="form"
      class="c-PJLV c-PJLV-ieHpPhp-css"
    >
      <div
        class="c-PJLV"
      >
        <div
          class="c-dhzjXW c-dhzjXW-iwHglP-css"
        >
          <label
            class="c-cVSpVL c-cVSpVL-gvmVBy-size-md c-cVSpVL-grKRUr-type-block"
            for="website"
          >
            Website
          </label>
        </div>
        <input
          class="c-fZTBsJ c-fZTBsJ-pHZal-size-md"
          id="website"
          name="website"
          placeholder="Website (e.g. https://google.com)"
          type="text"
        />
      </div>
      <div
        class="c-PJLV"
      >
        <div
          class="c-dhzjXW c-dhzjXW-iwHglP-css"
        >
          <label
            class="c-cVSpVL c-cVSpVL-gvmVBy-size-md c-cVSpVL-grKRUr-type-block"
            for="login"
          >
            Login
          </label>
        </div>
        <input
          class="c-fZTBsJ c-fZTBsJ-pHZal-size-md"
          id="login"
          name="login"
          placeholder="Login (e.g. you@example.com)"
          type="text"
        />
      </div>
      <button
        class="c-fkUJsw c-fkUJsw-elrwsr-size-md c-fkUJsw-gsycxF-cv"
        type="submit"
      >
        Submit
      </button>
    </form>
  </div>
</div>
`;
